testsuite/features/support/api_test.rb

```diff
@@ -50,8 +50,7 @@ def initialize(_host)
   attr_reader :schedule
   attr_reader :system
   attr_reader :user
-  attr_reader :token
-  attr_writer :token
+  attr_accessor :token
```

Comment:
`attr_accessor` combines both getter and setter I suppose?

Reply:
Yes.